kore/src/Kore/Internal/TermLike/TermLike.hs

```diff
@@ -948,9 +948,9 @@ traverseVariables adj termLike =
                 (TermLike variable1)
                 (RenamingT variable1 variable2 m (TermLike variable2))
         ->  RenamingT variable1 variable2 m (TermLike variable2)
-    worker (attrs :< termLikeF) = do
-        attrs' <- Attribute.traverseVariables askSomeVariableName attrs
-        let avoiding = freeVariables attrs'
+    worker (~attrs :< termLikeF) = do
```

Comment:
This is actually a [redundant](https://downloads.haskell.org/~ghc/latest/docs/html/users_guide/exts/strict.html#strict-by-default-pattern-bindings) (see "Pattern matching in case expressions..."):

```suggestion
    worker (attrs :< termLikeF) = do
```